addons/main/functions/fnc_terminateDevice.sqf

```diff
@@ -0,0 +1,62 @@
+/**
+ * PUBLIC
+ *
+ * This function handles the deletion of an AE3 asset. If the asset gets deleted for example by zeus, this function is triggered,
+ * but you can also trigger this function standalone.
+ * If applicable the function turns off the device and removes all power and network connections, both incoming and outgoing.
+ *
+ * Arguments:
+ * 1: Device <OBJECT>
+ *
+ * Results:
+ * None
+ *
+ * Example:
+ * [_device] call AE3_main_fnc_terminateDevice;
+ *
+ */
+
+params ["_device"];
+
+/* ================================================================================ */
+
+// turn off device
+[_device] call AE3_power_fnc_turnOffDevice;
```

Comment:
We probably want to check, if the device, is a power device first or?

Reply:
Yes, that's a good idea.